packages/cli/src/command-build.js

```diff
@@ -487,11 +499,12 @@ function makePackageBuild(
       packageInfo.directory,
     ))
 
-    packageHeader('building', packageInfo)
+    packageHeader('building', packageInfo, pointer, packagesLength)
```

Comment:
`packageIndex` would be a better name than `pointer`. Or maybe `buildIndex`?

Reply:
yap - changing